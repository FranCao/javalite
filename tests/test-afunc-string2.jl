int main()
{
    string s;
    string u;
    s = "Foo Foo Doo Bar";
    prints(s);
    u = upper(s);
    prints(u);
}