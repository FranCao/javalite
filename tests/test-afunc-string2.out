Foo Foo Doo Bar
FOO FOO DOO BAR